bigchaindb/commands/bigchain.py

```diff
@@ -11,7 +13,10 @@
 import bigchaindb
 import bigchaindb.config_utils
 from bigchaindb import db
-from bigchaindb.exceptions import DatabaseAlreadyExists, KeypairNotFoundException
+from bigchaindb.exceptions import (
```

Comment:
We should decide the format to handle multiple imports and add it to the style guide.
 I personally prefer
`from bigchaindb.exceptions import DatabaseAlreadyExists, KeypairNotFoundException`
since its less verbose and easier to read